src/Microsoft.Health.Fhir.Core/Resources.Designer.cs

```diff
@@ -816,16 +825,7 @@ internal class Resources {
                 return ResourceManager.GetString("SortNotSupported", resourceCulture);
             }
         }
-
-        /// <summary>
-        ///   Looks up a localized string similar to Include result was truncated.
-        /// </summary>
-        internal static string TruncatedIncludeMessage {
```

Comment:
>TruncatedIncludeMessage [](start = 31, length = 23)

@eladiw 
This resource was added in [https://github.com/microsoft/fhir-server/pull/1167](https://github.com/microsoft/fhir-server/pull/1167) but only autogenerated file was checked in without updating Resorces.resx.
I don't mind to update Resources.resx in this pr, but I'm not sure what should be the message.

Reply:
Since it has original message I added it to Resources.resx file

---
In reply to: [471665471](https://github.com/microsoft/fhir-server/pull/1195#discussion_r471665471) [](ancestors = 471665471)